### PR TITLE
Switched ADAL to common automation actions

### PR DESCRIPTION
**Pull request by oldalton**
Error conversion tests are failing, will remove work in progress tag once those are fixed.

### CODE REVIEW

File: ADAL/automation/actitons/ADAutoEmptyStessTestAction.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "ADAutoBaseAction.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ADAutoEmptyStessTestAction : ADAutoBaseAction
```
Comment (antrix1989) [inline ADAL/automation/actitons/ADAutoEmptyStessTestAction.h]: nit: 'r' missed

File: ADAL/automation/actitons/ADAutoEmptyStessTestAction.m [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    
+    __block BOOL stop = NO;
+    
+    while (!stop)
+    {
+        dispatch_semaphore_wait(sem, DISPATCH_TIME_FOREVER);
+        
+        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+            
+            NSError *contextError = nil;
```
Comment (jasoncoolmax) [inline ADAL/automation/actitons/ADAutoEmptyStessTestAction.m]: Is logging disabled when stress tests are run?

File: ADAL/automation/actitons/ADAutoExpireATAction.m [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+- (void)performActionWithParameters:(MSIDAutomationTestRequest *)parameters
+                containerController:(MSIDAutomationMainViewController *)containerController
+                    completionBlock:(MSIDAutoCompletionBlock)completionBlock
+{
+    id<ADTokenCacheDataSource> cache = [self cacheDatasource];
+
+    NSError *cacheError = nil;
+    
+    NSArray *cacheAliases = [self cacheAliasesWithParameters:parameters];
```
Comment (jasoncoolmax) [inline ADAL/automation/actitons/ADAutoExpireATAction.m]: Will it be more precise to be called cacheAuthorityAliases? :)

File: ADAL/automation/actitons/ADAutoInteractiveStressAction.m [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                      completionBlock:^(ADAuthenticationResult *result) {
+                          (void) result;
+                      }];
+}
+
+- (void)pollWithParameters:(MSIDAutomationTestRequest *)parameters
+       containerController:(MSIDAutomationMainViewController *)containerController
+           completionBlock:(MSIDAutoCompletionBlock)completionBlock
+{
+    [ADLogger setLevel:ADAL_LOG_LEVEL_NO_LOG];
```
Comment (jasoncoolmax) [inline ADAL/automation/actitons/ADAutoInteractiveStressAction.m]: :+1:

File: ADAL/automation/actitons/ADAutoNonEmptyStressAction.m [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+- (BOOL)needsRequestParameters
+{
+    return YES;
+}
+
+- (void)performActionWithParameters:(MSIDAutomationTestRequest *)parameters
+                containerController:(MSIDAutomationMainViewController *)containerController
+                    completionBlock:(MSIDAutoCompletionBlock)completionBlock
+{
```
Comment (jasoncoolmax) [inline ADAL/automation/actitons/ADAutoNonEmptyStressAction.m]: Better to run it with no logging? :)

**Pull request merged**